File: LeetCode/344_Reverse String.c
#include "stdio.h"

char* reverseString(char* s) {
    char temp[2];
	int i= 0,j=0;
    while(s[i] != '\0'){
		i++;
	}
	for(j;j<i;j++){
		i--;
		temp[1] = s[j];
		s[j] = s[i];
		s[i] = temp[1];
	}
	return s;
}

File: LeetCode/344_Reverse String.java
public class Solution {
    public String reverseString(String s) {
        char[] a = s.toCharArray();
        char b;
        for (int i=0,j=s.length()-1;i<s.length()/2;i++,j--){
            b = a[i];
            a[i] = a[j];
            a[j] = b;
        }
        return String.valueOf(a);
    }
}
